ui/less/range_elements.less

```diff
@@ -57,6 +57,11 @@
   /* The track should fill the range element. */
   width: 100%;
 
+ /* Since range elements are special input elements so they should reflect,
+   * user interaction so whhen user hover on range element the cusror should
+   * be pointer. */
+  cursor: pointer;
+  
```

Comment:
Hello @meet503186 this is already implemented in [PR](https://github.com/google/shaka-player/pull/3220/files)

Reply:
Ok then i close this PR